internal/pkg/cli/app_show.go

```diff
@@ -0,0 +1,172 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli/selector"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/config"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/describe"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
+	"github.com/spf13/cobra"
+)
+
+const (
+	appShowNamePrompt     = "Which application would you like to show?"
+	appShowNameHelpPrompt = "An application groups all of your services together."
```

Comment:
> An application is a collection of related services. 

What do you think about that?

Reply:
I think that's good! updated

Reply:
just a reminder if we decide to make this change then we need to change the other files for consistency

Reply:
I'll update the others in a separate PR